package.json

```diff
@@ -136,7 +136,9 @@
     "unit": "rm -f newrelic_agent.log && time tap --test-regex='(\\/|^test\\/unit\\/.*\\.test\\.js)$' --timeout=180 --no-coverage",
     "update-cross-agent-tests": "./bin/update-cats.sh",
     "versioned-tests": "./bin/run-versioned-tests.sh",
-    "versioned": "npm run prepare-test && time ./bin/run-versioned-tests.sh"
+    "update-changelog-version": "node ./bin/update-changelog-version",
+    "versioned": "npm run prepare-test && time ./bin/run-versioned-tests.sh",
```

Comment:
Should we differentiate between the `versioned` and `version` script a bit more? Maybe `version` could be `release-version`?

Reply:
I don't love how similar they are either.

That being said, I can't rename 'version' because that is a predefined script name by npm to get ran at the right time when `npm version` is called. There are pre, mid and post hooks. This one is mid.

We could rename the versioned test script but since a bunch of things are reliant on that I punted for now. Longer-term, perhaps we tweak the test side of things.

Reply:
That makes sense. Yeah, maybe we can rename versioned test script next time we need to do work on that.